easybuild/tools/systemtools.py

```diff
@@ -210,15 +291,37 @@ def get_cpu_model():
     os_type = get_os_type()
 
     if os_type == LINUX and os.path.exists(PROC_CPUINFO_FP):
-        # we need 'model name' on Linux/x86, but 'model' is there first with different info
-        # 'model name' is not there for Linux/POWER, but 'model' has the right info
-        model_regex = re.compile(r"^model(?:\s+name)?\s+:\s*(?P<model>.*[A-Za-z].+)\s*$", re.M)
         txt = read_file(PROC_CPUINFO_FP)
```

Comment:
The `txt` variable is actually used in both branches of the `if` statement...
But I'm fine with renaming.  Makes sense.